plugins/woocommerce/client/legacy/js/admin/settings.js

```diff
@@ -349,5 +349,30 @@
 			$underObservation.on( 'change', changeAgent );
 			changeAgent();
 		} );
+
+		(function () {
+			const activeTab = document.querySelector('.nav-tab-active');
+			if (!activeTab || window.innerWidth >= 500) return;
+
+			const parent = activeTab.parentElement;
+			if (!parent || parent.scrollWidth <= parent.clientWidth) return;
+
+			const tabLeft = activeTab.offsetLeft;
+			const tabRight = tabLeft + activeTab.offsetWidth;
+			const scrollLeft = parent.scrollLeft;
+			const visibleLeft = scrollLeft;
+			const visibleRight = scrollLeft + parent.clientWidth;
+
+			const isOutOfView = tabLeft < visibleLeft || tabRight > visibleRight;
+
+			if (isOutOfView) {
+				const offset = tabLeft - parent.clientWidth / 2 + activeTab.offsetWidth / 2;
+				parent.scrollTo({
+					left: offset,
+					behavior: 'auto'
+				});
+			}
+		})();
```

Comment:
The scroll-to-active-menu addition is a nice touch for UX!  I think it would be helpful to add some comments to the code.

Reply:
@chihsuan Updated it in [Fix the nav width when the setting page stretches beyond its parent w…](https://github.com/woocommerce/woocommerce/pull/56694/commits/ed1d071962d2270d66ddcbf2442767a8f6c96d22)